packages/wallet/frontend/src/components/userCards/CardButtonMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from '@/ui/Button'
+import { Eye, EyeCross, Snow, Trash } from '../icons/CardButtons'
+import { Dispatch, SetStateAction, useState } from 'react'
+import { CardTypes } from '@/pages/card'
+
+export const CardButtonMenu = ({
```

Comment:
Please add a TODO to revisit this when we have the final designs. I'm seeing a layout shift for the buttons when switching between different states of the card.

Reply:
Will add ToDO, but we will have no more final designs